etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamOrderSender.java

```diff
@@ -4,14 +4,8 @@
 import gov.hhs.cdc.trustedintermediary.etor.orders.Order;
 import gov.hhs.cdc.trustedintermediary.etor.orders.OrderSender;
 import gov.hhs.cdc.trustedintermediary.etor.orders.UnableToSendOrderException;
-import gov.hhs.cdc.trustedintermediary.wrappers.AuthEngine;
-import gov.hhs.cdc.trustedintermediary.wrappers.Cache;
-import gov.hhs.cdc.trustedintermediary.wrappers.HapiFhir;
-import gov.hhs.cdc.trustedintermediary.wrappers.HttpClient;
-import gov.hhs.cdc.trustedintermediary.wrappers.HttpClientException;
-import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
-import gov.hhs.cdc.trustedintermediary.wrappers.SecretRetrievalException;
-import gov.hhs.cdc.trustedintermediary.wrappers.Secrets;
+import gov.hhs.cdc.trustedintermediary.metadata.MetaDataStep;
+import gov.hhs.cdc.trustedintermediary.wrappers.*;
```

Comment:
Mind undoing this `*` import?  We like to have each import listed out.

Reply:
This has already been fixed.

Reply:
Yeah just did this

Reply:
done.

Reply:
FYI, here's how to disable the automatic wildcard import in IntelliJ: https://logfetch.com/java-intellij-no-wildcards/